app/src/main/res/layout/fragment_apps.xml

```diff
@@ -52,4 +52,15 @@
             android:scrollbars="vertical" />
     </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
 
+    <com.google.android.material.floatingactionbutton.ExtendedFloatingActionButton
+        android:id="@+id/updateReportsFAB"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentBottom="true"
+        android:layout_margin="16dp"
+        android:text="@string/refresh"
```

Comment:
Please remove text, it's take too space in screen
But add `android:contentDescription="@string/refresh"` for accessibility